container/monitor.go

```diff
@@ -244,11 +244,11 @@ func (m *containerMonitor) start() error {
 // previous runs success or failure.  Regardless of success, if the container had
 // an execution time of more than 10s then reset the timer back to the default
 func (m *containerMonitor) resetMonitor(successful bool) {
+	threshold := 60.0 // threshold is 60 seconds
 	executionTime := time.Now().Sub(m.lastStartTime).Seconds()
+	incrementTime := (time.Duration(m.timeIncrement) * time.Millisecond).Seconds()
 
-	if executionTime > 10 {
-		m.timeIncrement = defaultTimeIncrement
-	} else {
+	if executionTime < threshold && incrementTime < threshold {
```

Comment:
@WeiZhang555 I would keep the execution threshold to 10s though. It's the "maximum time to restart" that I think should be 60s :stuck_out_tongue_closed_eyes:.